capplet/gsm-app-dialog.c

```diff
@@ -414,30 +411,12 @@ gsm_app_dialog_class_init (GsmAppDialogClass *klass)
                                                               "comment",
                                                               NULL,
                                                               G_PARAM_READWRITE | G_PARAM_CONSTRUCT));
-
-        g_type_class_add_private (klass, sizeof (GsmAppDialogPrivate));
 }
 
 static void
 gsm_app_dialog_init (GsmAppDialog *dialog)
 {
```

Comment:
this function is empty, it can't be removed?

Reply:
This function is required for gobject system, it is instance initial function, called by other function.